jetty/release_notes.md

```diff
@@ -2,136 +2,318 @@
 
 # Release Notes
 
-## Zenoh in gz-transport
-
-Gazebo now supports Zenoh as an alternative transport implementation, offering
-improved discovery, interoperability, and performance. To enable Zenoh, set the
-environment variable `export GZ_TRANSPORT_IMPLEMENTATION=zenoh`. This allows
-Gazebo to leverage Zenoh's features and potentially integrate more seamlessly
-with ROS 2 and other systems utilizing Zenoh.
-
-See the [tracking issue](https://github.com/gazebosim/gz-transport/issues/559)
-and the implementation pull request
-<https://github.com/gazebosim/gz-transport/pull/665> for more details.
-
-## Improve Gazebo APIs for Reinforcement Learning
-
-Several APIs have been improved to streamline reinforcement learning pipelines.
-We have also added an [example integration with the StableBaselines3 Python package](https://github.com/gazebosim/gz-sim/tree/gz-sim10/examples/scripts/reinforcement_learning/simple_cart_pole)
-for reinforcement learning, enabling users to experiment with RL algorithms within
-Gazebo. The example provides a starting point for developing and testing robotic
-control policies.
-
-```{figure} https://github.com/user-attachments/assets/f30160a3-e04f-4ec1-aab4-111739b0d349
-:width: 400
-:alt: RL_with_gazebo_simple_example
-
-Example of doing Reinforcement Learning in Gazebo
+![Gazebo Jetty Artwork](img/rel_notes/jetty.png){w=500px h=500px}
```

Comment:
<img width="1080" height="1080" alt="Gazebo_Jetty_FINAL" src="https://github.com/user-attachments/assets/df73d95c-a7d0-459e-9e66-1625c99f2258" />

We color corrected the logo. This is the final version.

Reply:
Thanks. Fixed in https://github.com/gazebosim/docs/pull/636/commits/9c1d57727eaae120922dcaa3c4b7d3102ec04fb6